docs/api/extras/helpers/PolarGridHelper.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<base href="../../../" />
+		<script src="list.js"></script>
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		[page:Line] &rarr;
+
+		<h1>[name]</h1>
+
+		<div class="desc">The PolarGridHelper is an object to define polar grids. Grids are two-dimensional arrays of lines.</div>
+
+
+		<h2>Example</h2>
+
+		<code>var radius = 10;
+		var radials = 16;
+		var circles = 8;
+		var divisions = 50;
+
+		var helper = new THREE.PolarGridHelper( radius, radials, circles, divisions );
+		scene.add( helper );
+		</code>
+		[example:webgl_helpers Example using various helpers]
+
+
+		<h2>Constructor</h2>
+
+		<h3>[name]( [page:Number radius], [page:Number radials], [page:Number circles], [page:Number divisions], [page:Color color1], [page:Color color2] )</h3>
+		<div>
+		radius -- The radius of the polar grid. This can be any positive number. <br />
```

Comment:
I cribbed from GridHelper.html, but sure there's no reason we can't be _more_ informative since we're touching it :)